.github/workflows/hadolint-ci.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Hadolint Linting'
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: hadolint/hadolint-action@v3.1.0
+      with:
+        dockerfile: "Dockerfile*"
+        recursive: true
+        # https://github.com/hadolint/hadolint/wiki/DL3041
+        # https://github.com/hadolint/hadolint/wiki/DL3050
+        ignore: DL3041,DL3050
```

Comment:
Missing a newline here. Does your editor automatically remove these?

Reply:
Maybe cos it seems to work sometimes and doesn't. I'd have to review that and change it.